taxcalc/calcfunctions.py

```diff
@@ -3309,12 +3309,12 @@ def IITAX(c59660, c11070, c10960, personal_refundable_credit, ctc_new, rptc,
         Sum of iitax and payrolltax and lumpsum_tax
     """
     eitc = c59660
-    if CTC_refundable:
-        ctc_refund = c07220
+    if CTC_refundable and ODC_refundable:
```

Comment:
@bodiyang I would have a separate set of if/else cases for `ODC_refundable`, rather than combine as here.  What you've done misses the case of `CTC_refundable = False` and `ODC_refundable=True`.